recipes/cargo/cargo.inc

```diff
@@ -14,8 +14,12 @@ DEPENDS = "openssl zlib libgit2 curl ca-certificates libssh2"
 SRC_URI = "\
 	git://github.com/rust-lang/cargo.git;protocol=https;name=cargo \
 	git://github.com/rust-lang/rust-installer.git;protocol=https;name=rust-installer;destsuffix=git/src/rust-installer \
+	http://static-rust-lang-org.s3.amazonaws.com/cargo-dist/2015-04-02/cargo-nightly-x86_64-unknown-linux-gnu.tar.gz \
```

Comment:
Ah, I missed that this was for cargo. Nevermind, that seems ok (well, it only works for x86-64, but we can tweak that later).